adapters/Laravel/Vendor/tests/Feature/VendorRepositoryTest.php

```
@@ -0,0 +1,505 @@
+        $saved = $repo->save($tenantId, $this->makeVendor());
+
+        $found = $repo->findByTenantAndId($tenantId, (string) $saved->getId());
+
+        $this->assertNotNull($found);
+        $this->assertSame($saved->getId()->getValue(), $found->getId()->getValue());
+        $this->assertSame($saved->getLegalName()->getValue(), $found->getLegalName()->getValue());
+        $this->assertSame($saved->getDisplayName()->getValue(), $found->getDisplayName()->getValue());
+        $this->assertSame($saved->getRegistrationNumber()->getValue(), $found->getRegistrationNumber()->getValue());
+        $this->assertSame($saved->getCountryOfRegistration(), $found->getCountryOfRegistration());
+        $this->assertSame($saved->getStatus(), $found->getStatus());
+    }
```

PHPMD (2.15.0)

[error] The method test_save_and_fetch_vendor_by_tenant_and_id is not named in camelCase. (undefined)

(CamelCaseMethodName)